src/client/Client.cc

```diff
@@ -5933,6 +5933,11 @@ int Client::mds_check_access(std::string& path, const UserPerm& perms, int mask)
     }
   }
 
+  // drop any leading /
```

Comment:
I know @batrick proposed that this be a part of make_absolute_path_string(), but. this looks an appropriate place too. @batrick WDYT?

Reply:
I think make_absolute_path_string() should return path with leading single '/' as there might be other consumers of it in the future?